pfcwd/main.py

```diff
@@ -98,10 +100,16 @@ def get_server_facing_ports(db):
 
 
 class PfcwdCli(object):
-    def __init__(self, namespace=None, display=constants.DISPLAY_ALL):
+    def __init__(
+        self, ctx, db=None, namespace=None, display=constants.DISPLAY_ALL
+    ):
         self.db = None
         self.config_db = None
-        self.multi_asic = multi_asic_util.MultiAsic(display, namespace)
+        # ctx.obj contains the Db object set by clicommon.pass_db decorator
+        # Unit tests over write the ctx.obj with test Db object, using ctx.obj
```

Comment:
Do we really need to pass both ctx and db ? With the  @clicommon.pass_db decorator, the "db" should have the db object which we created and pass it into runner object in test scripts ?

Reply:
I'll test it out and find out if both are required. I expect "db" will be populated (with single ASIC) when the modules are loaded initially, before the masic env variable is set.